### DISCUSSION TITLE
모델링, 리팩터링 실습해보기

### DISCUSSION THREAD

**Discussion by corock** [cowaine/is-my-code-that-strange#118] [Chapter 13-14 (모델링, 리팩터링)] [answered]
- 모델링과 리팩터링을 간단하게 해볼 수 있도록 #115 을 작성했습니다.
- _starter-code_ 는 [DiceGameApplication](https://github.com/cowaine/is-my-code-that-strange/blob/develop/corock/src/main/java/com/cowaine/corock/chapter14/game/DiceGameApplication.java), [DiceGame](https://github.com/cowaine/is-my-code-that-strange/blob/develop/corock/src/main/java/com/cowaine/corock/chapter14/game/DiceGame.java) 총 2개의 클래스로 구현되어 있습니다.
- 위 코드를 보고 메서드 / 모델링 / 리팩터링 관점에서 개선할 부분이 있다면 댓글로 남겨주세요.

**Comment by firewoody237** [comment, marked as answer]
https://github.com/cowaine/is-my-code-that-strange/commit/fbe56d7399520371a2057807272a49578a9cac64
